packages/editor/src/components/editor-regions/index.js

```diff
@@ -18,10 +18,16 @@ function EditorRegions( {
 	className,
 } ) {
 	return (
-		<div className={ classnames( className, 'edit-post-editor-regions' ) }>
+		<div
+			className={ classnames(
+				className,
+				'components-editor-regions',
+				'edit-post-editor-regions'
+			) }
+		>
 			{ !! header && (
 				<div
-					className="edit-post-editor-regions__header"
+					className="components-editor-regions__header edit-post-editor-regions__header"
 					role="region"
 					/* translators: accessibility text for the top bar landmark region. */
 					aria-label={ __( 'Editor top bar' ) }
```

Comment:
I was wondering whether the aria-label are too opinionated and should be made customizable or not. Potentially, they couuld work for any "editor".

Reply:
Yeah, maybe :thinking: I'd go with YAGNI here -- don't make it customizable until needed?